docs/API_KEY.md

```diff
@@ -0,0 +1,59 @@
+# How to add new/own API key to tinode?
+
+## What is API key?
+
+Trinode encrypts data for security. For this purpose, two keys are used:

```

Comment:
Actually, these keys are not used for encryption. These keys are used to generate an API key which provides some minor protection from automated tools scraping the API.

Reply:
Also `Trinode`?